guice7/src/moditect/module-info.java

```diff
@@ -0,0 +1,14 @@
+module com.fasterxml.jackson.module.guice7 {
+    //Jakarta Reference Implementation
+    requires static jakarta.inject;
+
+    requires com.fasterxml.jackson.annotation;
```

Comment:
requires transitive

Reply:
transitive on  com.fasterxml.jackson.annotation?  ok, i'll defer to you on module-info stuff but want to be sure I get it right.  should that change also be made to the existing guice module?

Reply:
If change was made here, yes, I think similarly to existing. I don't have strong opinion.